strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/repository/NpmRepositoryFeatures.java

```diff
@@ -478,15 +480,14 @@ public void handle(RemoteRepositorySearchEvent event)
                 return;
             }
 
-            Predicate predicate = event.getPredicate();
```

Comment:
@anki2189 you shouldn't drop this `predicate` here. Anyway we should not proceed with this issue (see [comment]())